examples/pytorch_optimization.py

```diff
@@ -15,6 +15,8 @@
 
 import os
 import torch
+import matplotlib
+matplotlib.use('agg')
```

Comment:
In some machines, especially for servers when the DISPLAY environment not set, matplotlib library is likely to fail on those machines, as it sometimes intends to plot interactive figures. Adding this will make the matplotlib backend noninteractive, as we only want to generate a file call plot.png without user interface. Check more details on [Matplotlib Backend].(https://matplotlib.org/2.0.2/faq/usage_faq.html#what-is-a-backend)

Added one comment about this.